Gemfile.lock

```diff
@@ -1,84 +1,137 @@
+      connection_pool (>= 2.2.5)
+      drb
+      i18n (>= 1.6, < 2)
+      minitest (>= 5.1)
+      mutex_m
+      tzinfo (~> 2.0)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    adiwg-mdcodes (2.8.4)
+    adiwg-mdcodes (2.10.0)
       json (~> 2.0)
-    adiwg-mdjson_schemas (2.8.1)
-    builder (3.2.4)
+    adiwg-mdjson_schemas (2.9.5)
+    ast (2.4.2)
+    base64 (0.2.0)
+    bigdecimal (3.1.8)
+    builder (3.3.0)
     coderay (1.1.3)
-    concurrent-ruby (1.2.2)
+    concurrent-ruby (1.3.3)
+    connection_pool (2.4.1)
     crass (1.0.6)
-    erubi (1.12.0)
-    i18n (1.14.1)
+    debug (1.9.2)
+      irb (~> 1.10)
+      reline (>= 0.3.8)
+    drb (2.2.1)
+    erubi (1.13.0)
+    i18n (1.14.5)
       concurrent-ruby (~> 1.0)
-    jbuilder (2.11.5)
+    io-console (0.7.2)
+    irb (1.14.0)
+      rdoc (>= 4.0.0)
+      reline (>= 0.4.2)
+    jbuilder (2.12.0)
       actionview (>= 5.0.0)
       activesupport (>= 5.0.0)
-    json (2.6.3)
+    json (2.7.2)
     json-schema (2.8.1)
       addressable (>= 2.4)
     kramdown (2.4.0)
       rexml
-    loofah (2.21.1)
+    language_server-protocol (3.17.0.3)
+    loofah (2.22.0)
       crass (~> 1.0.2)
-      nokogiri (>= 1.5.9)
-    mini_portile2 (2.4.0)
-    minitest (5.20.0)
-    nokogiri (1.10.10)
-      mini_portile2 (~> 2.4.0)
-    public_suffix (4.0.7)
-    rails-dom-testing (2.1.1)
+      nokogiri (>= 1.12.0)
+    minitest (5.23.1)
+    mutex_m (0.2.0)
+    nokogiri (1.16.6-arm64-darwin)
+      racc (~> 1.4)
+    nokogiri (1.16.6-x86_64-darwin)
+      racc (~> 1.4)
+    parallel (1.25.1)
+    parser (3.3.3.0)
+      ast (~> 2.4.1)
+      racc
+    psych (5.1.2)
+      stringio
+    public_suffix (5.1.1)
+    racc (1.8.0)
+    rails-dom-testing (2.2.0)
       activesupport (>= 5.0.0)
       minitest
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.5.0)
-      loofah (~> 2.19, >= 2.19.1)
-    rake (13.1.0)
-    rexml (3.2.5)
+    rails-html-sanitizer (1.6.0)
+      loofah (~> 2.21)
+      nokogiri (~> 1.14)
+    rainbow (3.1.1)
+    rake (13.2.1)
+    rdoc (6.7.0)
+      psych (>= 4.0.0)
+    regexp_parser (2.9.2)
+    reline (0.5.9)
+      io-console (~> 0.5)
+    rexml (3.3.0)
+      strscan
+    rubocop (1.64.1)
+      json (~> 2.3)
+      language_server-protocol (>= 3.17.0)
+      parallel (~> 1.10)
+      parser (>= 3.3.0.2)
+      rainbow (>= 2.2.2, < 4.0)
+      regexp_parser (>= 1.8, < 3.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.31.1, < 2.0)
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (>= 2.4.0, < 3.0)
+    rubocop-ast (1.31.3)
+      parser (>= 3.3.1.0)
+    ruby-progressbar (1.13.0)
+    stringio (3.1.1)
+    strscan (3.1.0)
     thor (0.20.3)
-    thread_safe (0.3.6)
-    tzinfo (1.2.11)
-      thread_safe (~> 0.1)
+    tzinfo (2.0.6)
+      concurrent-ruby (~> 1.0)
+    unicode-display_width (2.5.0)
     uuidtools (2.2.0)
 
 PLATFORMS
-  arm64-darwin-22
-  x86_64-linux
+  arm64-darwin-23
+  x86_64-darwin-23
```

Comment:
we might need to re-add the `x86_64-linux` platform back in to get it build in CI.